energy-integration/mex-governance/src/errors.rs

```diff
@@ -0,0 +1,8 @@
+pub const EMISSION_RATE_ZERO: &[u8] = b"Emission rate cannot be zero";
+pub const WEEK_ALREADY_INITIALIZED: &[u8] = b"Week already initialized";
+pub const ALREADY_VOTED_THIS_WEEK: &[u8] = b"Already voted this week";
+pub const INVALID_VOTE_AMOUNT: &[u8] = b"Invalid vote amount";
+pub const FARM_ADDRESS_NOT_WHITELISTED: &[u8] = b"Farm address is not whitelisted";
+pub const INVALID_FARM_ADDRESS: &[u8] = b"Invalid farm address";
+pub const FARM_NOT_WHITELISTED: &[u8] = b"Farm is not whitelisted";
+pub const FARM_BLACKLISTED: &[u8] = b"Farm is blacklisted";
```

Comment:
Statics behave better than consts for strings.

```suggestion
pub static EMISSION_RATE_ZERO: &[u8] = b"Emission rate cannot be zero";
pub static WEEK_ALREADY_INITIALIZED: &[u8] = b"Week already initialized";
pub static ALREADY_VOTED_THIS_WEEK: &[u8] = b"Already voted this week";
pub static INVALID_VOTE_AMOUNT: &[u8] = b"Invalid vote amount";
pub static FARM_ADDRESS_NOT_WHITELISTED: &[u8] = b"Farm address is not whitelisted";
pub static INVALID_FARM_ADDRESS: &[u8] = b"Invalid farm address";
pub static FARM_NOT_WHITELISTED: &[u8] = b"Farm is not whitelisted";
pub static FARM_BLACKLISTED: &[u8] = b"Farm is blacklisted";
```

Reply:
Done.